scripts/pylib/twister/twisterlib/jobserver.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2022 The ChromiumOS Authors
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This needs to be cleaned up.  Zephyr's LICENSE file is specifically Apache-2.0, while the [Chromium LICENSE](https://source.chromium.org/chromiumos/chromiumos/codesearch/+/HEAD:src/platform/ec/LICENSE) file is the BSD-style license.
This probably should just be `SPDX-License-Identifier: BSD-3-Clause`.

Reply:
Embedding the Chromium LICENSE directly might be appropriate, similar to [strtoul.c](https://github.com/zephyrproject-rtos/zephyr/blob/main/lib/libc/minimal/source/stdlib/strtoul.c).  But I'm not a lawyer.

Reply:
done

Reply:
Google lawyers said to use 

```
# Copyright 2022 Google LLC.
# SPDX-License-Identifier: Apache-2.0
```

So that is better for Zephyr also.